pkg/csi/service/vanilla/controller.go

```diff
@@ -1255,6 +1267,12 @@ func (c *controller) ControllerUnpublishVolume(ctx context.Context, req *csi.Con
 			return nil, csifault.CSIInvalidArgumentFault, logger.LogNewErrorCodef(log, codes.Internal,
 				"validation for UnpublishVolume Request: %+v has failed. Error: %v", *req, err)
 		}
+
+		volumeManager, err := getVolumeManagerForVolumeID(ctx, c, req.VolumeId, volumeInfoService)
+		if err != nil {
+			return nil, csifault.CSIInvalidArgumentFault, logger.LogNewErrorCodef(log, codes.Internal,
```

Comment:
Can we use `csifault.CSIInternalFault` instead?

Reply:
done.